flask_sqlalchemy/__init__.py

```diff
@@ -390,16 +390,28 @@ def iter_pages(self, left_edge=2, left_current=2,
               </div>
             {% endmacro %}
         """
+
+        pages = []
+        pages += range(
```

Comment:
This breaks for Python 3. Why not `pages.extend` or  just `itertools.chain(...)` since there's not a reason to keep them all in memory anyways.